content/en/security/application_security/enabling/compatibility/serverless.md

```diff
@@ -37,33 +55,44 @@ Go
 .NET
 : 
 
-### ASM capabilities 
-The following ASM capabilities are not supported for Lambda functions:
- - ASM Risk Management
- - IP, user, and suspicious request blocking
- - 1-Click enabling ASM
 
-## ASM capabilities support
+### Google Cloud Run
+Notes
+- Google Cloud Run support is in public beta for ASM 
+- Threat Protection via Remote Config is not supported. However you can use Workflows to block IPs in your WAF instead
+   
+|Type           	| Threat Detection	|  Vulnerability Management for OSS	| Vulnerability Management for Code-Level 	|
+| ---  		|   ---             		|           ----           			|           ----            				|
+| Java  		| beta         	| 	                			|						|
+| .NET    	| beta         	|  	                          		|						|
+| Node 		| beta     		| beta	              			|						|
+| Python   	| beta         	| beta                 			|						|
+| Ruby   	| beta         	|  	                 			|						|
+| PHP   	| 		        	|	            			|						|
+| Go   		| beta         	| 	               			|						|
 
-The following ASM capabilities are supported for serverless, for the specified Datadog Lambda extension version:
+Get started with ASM monitored on your services →
```

Comment:
```suggestion
Get started with ASM monitoring on your services →
```

Reply:
same with this one, removed this sentence here